### ISSUE TITLE
Project Plan Specification 7-9

### ISSUE THREAD

**Issue by ambergarcia** [G915-2-Data-Modeling-381/Project-1-eStreamingMovies#41]
Create a description up 2 - 6 sentences long for each number assigned.

**Comment by ambergarcia** [comment]
7. The project implementation will work incrementally starting with the Conceptual Data Model. It’s important to decide the business objects and business rules before continuing development, hence we’ll work incrementally and return to previous iterations if modifications are required. Each release is based on the model that is being worked on, for example while working on the Conceptual Data Model for eStreamingMoviesSSMS2019 the epic itself will have different drafts and stories that the team members will complete until its scheduled release date. If more time is required, we’ll postpone but continue to work on details not required for the release. In regards to specific documentation for the models and the presentation, those will be done all at once since each individual’s work is reliant on it.

8. The plan for our proposed solution is split into four sections; The Conceptual, Logical, and Physical Data Model development are different linear models that require the prior model to be completed before continuing. This means more time will need to spent deliberating on the logical concepts for our models (between concept and logical), and then the bottom line in our physical. Once this is complete only then can we write proper documentation for the project and go forward with working on our presented proposal of the completed work done towards the solution for the business. To meet our goals, parts of the models and presentation work are divided equally between group members, and we keep track of all issues assigned via GitHub. 

9. The overall project is going to be delivered before the due date, but the CDM/LDM/PDM will be completed a day before the team members work on the presentation. The highest-level timeline/schedule that has been decided via milestones for our project is as follows; Project Analysis by 10/1/2022 Completed CDM by 10/3/2022, Completed LDM by 10/5/2022, Completed PDM by 10/7/2022, and final presentation completed by 10/9/2022. Adjustments are allowed in between each section, if necessary, but it’s expected to be completed exactly on 10/9/2022 regardless of incoming delays.
